### ISSUE TITLE
Failed to reproduce results on Atari Pong

### ISSUE THREAD

**Issue by anonymous14159** [jsikyoon/dreamer-torch#7]
Hi, 
thank you very much for your contribution of pytorch implementation of Dreamer-v2!
However, I found it hard to reproduce the results on Atari Pong. I noticed previous discussion in issue  #2  and tried configurations with both precision 32 bits and 16 bits. However, I still failed to use the latest code to reproduce the results you have provided and could only get about -6 as maximum reward after training with several different seeds. The following is my training curve.
So I was wondering whether there is a configuration or version of code that can work for Atari Pong.
<img width="344" alt="image" src="https://user-images.githubusercontent.com/103020628/161791428-65cca97a-a44a-4b35-9a09-69a3819a49f4.png">

Thank you very much for your effort.
Best regards.

**Comment by jsikyoon** [comment]
Hi xmzhu00, sorry for the late reply. I will try to reproduce the Pong result again.

**Comment by jsikyoon** [comment]
Hi xmzhu00,

I also checked we cannot reproduce the performance as I reported before,

<img width="728" alt="image" src="https://user-images.githubusercontent.com/22428579/163735943-fe1de1b7-79e9-462c-aea3-3b1f38a80811.png">

I will keep checking it and share.

Thank you for pointing out it.

Bests, Jaesik.

**Comment by anonymous14159** [comment]
Thank you for your reply!
Looking foward to your further results.

**Comment by jsikyoon** [comment]
I ran with several random seeds and can find sometimes it is working slower, but usually, it is fine.

<img width="703" alt="image" src="https://user-images.githubusercontent.com/22428579/164580575-64f27326-0c4e-493a-ae85-430958320bc1.png">

Please check with multiple seeds to more concretely validate :)

**Comment by jsikyoon** [comment]
I think that this issue is addressed, so I will close it.

If you have more concerns, please feel free open a new issue.
